src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,13 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    public String getPrintableString(Printable... printables){
```

Comment:
Correct implementation of the vararg method. As discussed in tutorial, there are many suitable places for this method as it is very general.